### PR TITLE
Documentation warnings fixes

### PR DESCRIPTION
**Pull request by vivien-apple**
#### Problem

Fix some documentation warnings that appears in Xcode.

### CODE REVIEW

File: src/transport/SecureSession.h [hunk 1]
```diff
@@ -60,10 +62,10 @@ class DLL_EXPORT SecureSession
      *   Derive a shared key. The derived key will be used for encryting/decrypting
      *   data exchanged on the secure channel.
      *
-     * @param remote_public_key  A pointer to peer's public key
-     * @param public_key_length  Length of remote_public_key
-     * @param local_private_key  A pointer to local private key
-     * @param private_key_length Length of local_private_key
+     * @param secret             A pointer to the share secret
```
Comment (sagar-apple) [inline src/transport/SecureSession.h:65]: this should be "the shared secret" right?

**Review by sagar-apple — APPROVED** [review]
minor nit but LGTM otherwise.

**Review by saurabhst — APPROVED** [review]

File: src/transport/SecureSession.h (hunk 1 above)
Comment (vivien-apple) [reply src/transport/SecureSession.h]: Thanks ! Fixed.

File: src/darwin/Framework/CHIP/CHIPDeviceController.mm [hunk 2]
```diff
@@ -396,7 +396,7 @@ + (NSString *)commandToString:(NSData * _Nonnull)response
     uint8_t * bytes = (uint8_t *) response.bytes;
 
     EmberApsFrame frame;
-    if (extractApsFrame(bytes, response.length, &frame) == 0) {
+    if (extractApsFrame(bytes, (uint32_t) response.length, &frame) == 0) {
```
Comment (mspang) [inline src/darwin/Framework/CHIP/CHIPDeviceController.mm:399]: static_cast ?

**Review by mspang — APPROVED** [review]

**Review by rwalker-apple — APPROVED** [review]

**Pull request merged**